md/api-extensions.md

```diff
@@ -31,6 +31,10 @@ REST API extensions use the same authorization mechanism as the standard Bonita
 **Warning**: If you are running Bonita BPM in a cluster, after you import a REST API extension, you must restart the application server on all the cluster nodes.
 :::
 
+::: warning
+**Warning**: As the deployment of the REST API extensions automatically create the appropriate resource permission mappings, be careful to not [push configuration](BonitaBPM_platform_setup.md#update_platform_conf) without doing a pull previously. This would lost the running configuration.
```

Comment:
```suggestion
**Warning**: As the deployment of the REST API extensions automatically creates the appropriate resource permissions mapping, be careful not to [push a configuration](BonitaBPM_platform_setup.md#update_platform_conf) that has been updated before the deployment. This would overwrite the deployed configuration. To avoid doing so, pull the latest configuration, edit the needed elements and push it back to the Bonita runtime in a pretty short amount of time.
```

Reply:
:warning: @NathalieC for me, introducing `before the deployment` is unclear and would introduce confusion. Do you mean something like `before the push` ?

Reply:
Yes. Before the push. Sorry for the confusion.